### PR TITLE
Organisation dashboard

### PR DESCRIPTION
**Pull request by gov-cjwaszczuk**
## Summary
PR to add an organisation's services to it's frontpage / dashboard, also adds team members to the sidebar with a static page for the time being. Also has a few changes to link services within it. If there is no organisations, the page will convey this, and not allow the user to try and post. If the organisation isn't set on the service, the 'Not Set' will now be grey rather than black.
## Screenshots 
image
image

### CODE REVIEW

File: app/templates/org_nav.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<nav class="navigation">
+  <ul>
+    {% if current_user.has_permissions(admin_override=True) %}
```
Comment (quis) [inline app/templates/org_nav.html]: Can you add a ‘Dashboard’ link here are well? Should link to the list of services for that org.

File: app/templates/org_template.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+{% block fullwidth_content %}
+  <div id="content">
+    <div class="navigation-service">
+      <div class="navigation-service-name">
+        {{ current_org.name }}
+      </div>
+      <a href="{{ url_for('main.choose_service') }}" class="navigation-service-switch">Switch service</a>
+    </div>
+    <div class="grid-row">
+      {% if help %}
```
Comment (quis) [inline app/templates/org_template.html]: Don’t need the `help` stuff here – it’s only for new users creating their first service.

File: app/main/__init__.py [hunk 3]
```diff
@@ -3,34 +3,34 @@
...
+    register,
+    send,
+    service_settings,
+    sign_in,
+    sign_out,
+    styleguide,
+    templates,
+    two_factor,
+    user_profile,
+    verify
```
Comment (quis) [inline app/main/__init__.py]: Ideally this kind of rearranging things would be better done as a separate commit or PR. It makes it harder to see what’s going on when it’s mixed into functional changes.

If alphabetical is what we want then it would be better to change it everywhere, maybe using a tool like [isort](https://pypi.python.org/pypi/isort), and then linting for violations.

File: app/templates/views/organisations/organisation/index.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{% extends "org_template.html" %}
+
+{% block per_page_title %}
+  {{ current_org.name }}
+{% endblock %}
+
+{% block service_page_title %}
+  {{ current_org.name }}
+{% endblock %}
+
```
Comment (quis) [inline app/templates/views/organisations/organisation/index.html]: You shouldn’t have `per_page_title` and `service_page_title`. One overrides the other.

File: app/templates/views/organisations/organisation/update-organisation.html [hunk 5]
```diff
@@ -3,16 +3,16 @@
...
 
 {% block per_page_title %}
-  {{ '{} an organisation'.format('Update' if organisation else 'Create')}}
+  Update an organisation
 {% endblock %}
 
 {% block service_page_title %}
-  {{ '{} an organisation'.format('Update' if organisation else 'Create')}}
+  Update an organisation
 {% endblock %}
```
Comment (quis) [inline app/templates/views/organisations/organisation/update-organisation.html]: Same thing with `service_page_title` and `per_page_title` here.

File: app/templates/views/organisations/organisation/users/index.html [hunk 6]
```diff
@@ -0,0 +1,17 @@
+{% extends "org_template.html" %}
+
+{% block service_page_title %}
+  Team members
+{% endblock %}
```
Comment (quis) [inline app/templates/views/organisations/organisation/users/index.html]: Actually is it worth having a separate thing called `org_page_title`, since this is not a page for a service?

File: app/templates/views/service-settings.html [hunk 7]
```diff
@@ -257,7 +256,9 @@ <h2 class="heading-medium">Platform admin settings</h2>
       ) %}
         {% call row() %}
           {{ text_field('Organisation')}}
-          {{ optional_text_field(organisation or 'Not Set') }}
+          {% call field(status='default' if not organisation else '') %}
+              {{ organisation or 'Not Set' }}
+          {% endcall %}
```
Comment (quis) [inline app/templates/views/service-settings.html]: Why doesn’t the `optional_text_macro` work here? Either way it should be _Not set_ not _Not Set_.

File: tests/app/main/views/test_service_settings.py [hunk 8]
```diff
@@ -2037,6 +2037,25 @@ def test_select_organisation(
...
+        url_for('.link_service_to_organisation', service_id=service_one['id']),
+    )
+
+    assert response.status_code == 200
+    page = BeautifulSoup(response.data.decode('utf-8'), 'html.parser')
+
+    assert normalize_spaces(page.select_one('main p').text) == "No organisations"
+    assert not page.select_one('main button')
+
+
```
Comment (quis) [inline tests/app/main/views/test_service_settings.py]: Need a test to check that the organisations are shown if the API returns them.

Comment (gov-cjwaszczuk) [reply tests/app/main/views/test_service_settings.py]: Is this not covered in test_should_show_overview?

Comment (quis) [reply tests/app/main/views/test_service_settings.py]: Yeah, my bad, didn’t realise this code was already in.
